bin/ch-run.c

```diff
@@ -274,6 +274,9 @@ void fix_environment(struct args *args)
       INFO("new $PATH: %s", new_value);
    }
 
+   // $TMPDIR: Unset.
+   Z_ (unsetenv("TMPDIR"));
```

Comment:
Why are you unsetting $TMPDIR before `containerize()`?

Reply:
The justification is in `ch-run_desc.rst` though that's of course open to critique.

Note this makes the `host_tmp()` function approach fail, so see the latest commit. Sorry for the churn.